libraries/botbuilder-ai/src/qnaMaker.ts

```diff
@@ -9,7 +9,7 @@ import { Activity, TurnContext, BotTelemetryClient, NullTelemetryClient } from '
 import * as entities from 'html-entities';
 import * as os from 'os';
 const pjson: any = require('../package.json');
-import * as request from 'request-promise-native';
+const request = (new Function('require', 'if (!this.hasOwnProperty("fetch")) { return require("node-fetch"); } else { return this.fetch; }'))(require);
```

Comment:
Shouldn't we rename the `request` variable into something like `fetch`?

Reply:
I tried to leave it as close as possible to the original code to avoid confusion.